### PR TITLE
feat: add new category, migrations

### PR DESCRIPTION
**Pull request by IndexXuan**
### Why
- it is important and helpful to beginners, how to migration from other tools like vue-cli/create-react-app and so on.
- maybe something like cra-to-vite.

### Checklist

- [x] Title as described.
- [x] Make sure you put things in the right category.
- [x] The description of your item should be a sentence with less than 24 words.
- [x] Omit unnecessary words already provided in the context (e.g. `for Vite`, `a Vite plugin`).
- [x] Always add your items to the end of a list.

### Plugins/Tools

- [ ] The plugin/tool is working with **Vite 2.0 and onward**.
- [ ] The project is Open Source.
- [ ] The project follows the [Vite Plugins Conventions](https://vitejs.dev/guide/api-plugin.html#conventions).
- [ ] The repo should be at least 30 days old.
- [ ] The documentation is in English.
- [ ] The project is active and maintained.
- [ ] The project accepts contributions.
- [ ] Not a commercial product.

### Apps/Websites

- [ ] The website is available without errors and load in a reasonable amount of time.
- [ ] The website is using Vite intensively.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,6 +276,12 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 - [vite-plugin-ruby](https://github.com/ElMassimo/vite_rails/tree/main/package) - Vite configuration for Ruby backends.
 - [vite_rails](https://github.com/ElMassimo/vite_rails) - Manage your Rails assets with Vite.
 
+## Migration from others
+
+### vue-cli
+
+- [vue-cli-plugin-vue](https://github.com/IndexXuan/vue-cli-plugin-vite) - Use vite today, with vue-cli with minimize codebase modifications.
```
Comment (antfu) [inline README.md:283]:
```suggestion
## Migrations

### Vue CLI

- [vue-cli-plugin-vue](https://github.com/IndexXuan/vue-cli-plugin-vite) - Use Vite on Vue CLI with minimize codebase modifications.
```

**Review by Co0olboi — APPROVED** [review]

**Comment by IndexXuan** [comment]
@antfu done. BTW, npm scripts toc use  `"npx markdown-toc -i --bullets=- README.md",`, but no deps or devDeps have markdown-toc, how it works ? am i wrong ?
If you forget it, I can add the deps.

**Comment by antfu** [comment]
No, I omitted the deps intentionally. npx will install them whenever necessary, so we don't need to maintain local deps and a lock. Thanks for asking

**Comment by IndexXuan** [comment]
@antfu when yarn toc will invoke ? I didn't see anywhere, I think it should be in pre-commit or something ?

**Comment by antfu** [comment]
I do it manually, it's not a big deal.

**Pull request merged**